src/config/logger.js

```diff
@@ -0,0 +1,26 @@
+const winston = require('winston')
+
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.json(),
+  transports: [
+    new winston.transports.File({ filename: 'logs/error.log', level: 'error', handleExceptions: true }),
```

Comment:
If DevOps guys want to collect the app from 5 app instance, what should we do?

Reply:
I don’t understand your question, can you please clarify a bit

Reply:
Never mind,  It's not a good question.

Reply:
ah ok , no worries